docs/q1_report.md

```diff
@@ -0,0 +1,81 @@
+# Quick Glance at Q1
+
+Metrics on the Operate First Data Science Community Meetup between November 16, 2021 - March 31, 2022
+
+* [Detailed dashboard](https://datastudio.google.com/u/1/reporting/de85ebeb-c76a-4930-85bf-b07a1d2becd3/page/qiqpB/edit)
+* [Past meetups and recordings](content.md)
+
+
+### Aim of the community 
+
+The Operate First Data Science Community Meetup is an event open to everyone interested in learning more about how various individuals, teams, and online communities are working towards democratizing the development of production-level AI. 
+
+This meetup aims to embrace the multi-disciplinary nature of the current state of AI operations and cloud infrastructure by covering technical topics relevant to data scientists, software engineers, DevOps professionals, and statisticians alike. And all in an open public forum.
+
+
+### Number of Meetups hosted 
+
+8
+
+### Total Number of Attendees
+
+155
+
+### Total Number of Unique Attendees
+
+74
+
+### Average number of Attendees per Meetup
+
+28
+
+
+### Internal vs external attendees
+
+
+![image 1](assets/image1.png)
+
+### Number of attendees for each meetup 
+
+**Consistent attendance across meetups with the lowest number of attendees being 21**
```

Comment:
This was also the most recent meetup. Do you want to point out the downward trend, and not something like "We've noticed this downward trend in attendance over last few meetups. Suggestions for adressing this issue are outlined below" or something?

Reply:
Okay, sounds good. We will add that